### PR TITLE
[experimental] Implement one-helper, "ping-pong" aggregation.

### PR DESCRIPTION
**Pull request by branlwyd**
This is an experimental change following https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/393.

Other than specifying to one helper & implementing ping-ponging, there are a few related changes as well:

 * I enabled the use of Poplar1 to test aggregation-continuation behavior. Neither Prio3 nor the dummy VDAF would work for this, as they do not have enough rounds to trigger continuation. (In retrospect, it may have been wiser to extend the dummy VDAF to allow the number of rounds to be controlled.)

### CODE REVIEW

**Comment by tgeoghegan** [comment]
I think it'd help reviewers to rebase this on #1253.

**Comment by branlwyd** [comment]
> I think it'd help reviewers to rebase this on #1253.

Done.

**Review by tgeoghegan — COMMENTED** [review]
This looks like a sound implementation of https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/393 to me.

File: aggregator/src/aggregator/http_handlers.rs [hunk 1]
```diff
@@ -867,26 +869,6 @@ mod tests {
         )
         .await;
 
-        // should reject a report with only one share with the unrecognizedMessage type.
```
[the remark this answers is not in the document]
Comment (branlwyd) [reply aggregator/src/aggregator/http_handlers.rs:870]: Yup -- that's one of the nice things about switching to leader/helper-specific fields. This is another advantage of specifying to exactly one helper.

**Comment by tgeoghegan** [comment]
Assigning to myself: I'll rebase this and sync it up with what got published in DAP-05.

**Comment by tgeoghegan** [comment]
Ping pong was landed in #1811, closing this PR.

**Pull request closed without merging**